pkg/plugins/mellanox/mellanox_plugin.go

```diff
@@ -215,6 +221,20 @@ func configFW() error {
 	return nil
 }
 
+func resetFW() error {
+	glog.Info("mellanox-plugin resetFW()")
+	for pciAddr := range attributesToChange {
+		cmdArgs := []string{"-d", pciAddr, "-y", "reset"}
+
+		_, err := utils.RunCommand("mstfwreset", cmdArgs...)
```

Comment:
This will print the stdout of mstfwreset right? If not we may want something like `glog.V(2).Infof("mellanox-plugin: configFW(): %v", cmdArgs)` from above.

Reply:
Yes utils.RunCommand will print the stdout of mstfwreset, but I also added this additional log consistent with configFW().